test/canAddLikeTermPolynomialNodes.test.js

```diff
@@ -0,0 +1,16 @@
+const canAddLikeTermPolynomialNodes = require('../lib/checks/canAddLikeTermPolynomialNodes');
+
+const TestUtil = require('./TestUtil');
+
+function canBeAdded(expr, addable) {
+  TestUtil.testBooleanFunction(canAddLikeTermPolynomialNodes, expr, addable);
+}
+
+describe('Add like term polynomials', () => {
+  const tests = [
+    ['x^2 + x^2', true],
+    ['x + x', true],
+    ['x^3 + x', false],
+  ];
+  tests.forEach(t => canBeAdded(t[0], t[1]));
```

Comment:
oh sorry I missed these before - they should be called "testCanBeAdded" like in the other test files

and similar for the other test functions